src/main/java/mat/server/service/impl/MatContextServiceUtil.java

```diff
@@ -184,17 +185,32 @@ public boolean isCurrentCQLLibraryDraftable(CQLLibraryDAO cqlLibraryDAO,
         return isCurrentCQLLibraryEditable(cqlLibraryDAO, libraryId, false);
     }
 
+    public boolean isValidatable(Measure measure) {
+        return isFhirMeasureValidatabe(measure) || isQdmMeasureValidatable(measure);

```

Comment:
Minor thing. Both sides will execute whenever it is a QDM measure. Maybe replace with either an if check on measure.isFhirMeasure() or a ternary operator.

Reply:
yeah. it will. should not affect performance. it's not expensive.
not sure what solution you proposed. could you clarify?

like   return (measure.isFhir() && isFhirValidatable(measure)) || isQdmMeasureValidatable(measure)

but that will make code look less readable.. with minimum performance gains

Reply:
Something like:
measure.getMeasureModelModel() == FHIR ? isFhirMeasureVAlidatable(measure) : isQdmMeasureValidatable(measure);

Reply:
```
if (measure.isFhirMeasure()) {
  return isFhirMeasureValidatabe(measure);
}
return isQdmMeasureValidatable(measure);
```
Or
`return measure.isFhirMeasure() ? isFhirMeasureValidatabe(measure) : isQdmMeasureValidatable(measure);`

Reply:
ok

Reply:
well, I would really question any performance gains from that.
And I think that boolean expression is preferred to if/ternary one. 

@cday-semanticbits & @jkotanchik-SB 
How about measure.isFhirMeasure() && measure.isDraft() || isQdmMeasureValidatable(measure)

Reply:
return measure.getMeasureModel() == FHIR ? isFhirMeasureVAlidatable(measure) : isQdmMeasureValidatable(measure); seems right to me but haven't looked at it closely